www/contribute/index.html

```diff
@@ -22,6 +22,7 @@
             <li>Join the <a href="{{ site.baseurl }}/contact">Dev mailing list</a> and send a brief introduction of yourself to it</li>
             <li>Join the discussion on <a href="http://slack.cordova.io/">Slack</a></li>
             <li>Sign the <a href="http://www.apache.org/licenses/#clas">Individual Contributor License Agreement (ICLA)</a> and <a href="mailto:secretary@apache.org">submit it</a></li>
+            <li>Try out the next version of Cordova using <a href="./nightly_builds.html">nightly builds</a></li>
```

Comment:
Maybe use an absolute link like `{{ site.baseurl }}/contribute/nightly_builds.html` here, in case this page ever moves.